packages/dev/core/src/Shaders/iblDominantDirection.fragment.fx

```diff
@@ -0,0 +1,27 @@
+precision highp sampler2D;
+precision highp samplerCube;
+
+#include<helperFunctions>
+#include<importanceSampling>
+#include<pbrBRDFFunctions>
+#include<hdrFilteringFunctions>
```

Comment:
I think you only need the `hdrFilteringFunctions` include.

Reply:
If any one of these is missing, it doesn't compile.

Reply:
My bad, the functions from `hdrFilteringFunctions` that we don't use still need these includes...